src/validate_logic.ts

```diff
@@ -60,7 +81,31 @@ function validateUniqueness(networks: Network[]) {
       .filter((v, i) => values.indexOf(v) !== i)
       .filter((v) => !ALLOWED_DUPLICATES.includes(v));
     if (duplicates.length) {
-      ERRORS.push(`Duplicate field: "${field} = ${duplicates[0]}"`);
+      // Find all networks with the duplicate value
+      const duplicateValue = duplicates[0];
```

Comment:
For simplicity, let's just allow duplicates for chains linked with `evmOf` relation instead